build.gradle

```diff
@@ -39,6 +39,7 @@ subprojects {
         codeVersion = 1
 
         hyperwalletCoreVersion = '1.0.0-beta05-SNAPSHOT'
+        hyperwalletInsightVersion = '1.0.0-beta01-SNAPSHOT'
```

Comment:
beta02 is being worked on as we speak to fix some of the bugs found. Will leave this as beta01 for now to hopefully get maven central to pick it up.